types/microrouter/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for microrouter 2.2
+// Project: https://github.com/pedronauck/micro-router#readme
+// Definitions by: Mathieu Dutour <https://github.com/mathieudutour>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node"/>
+
+import { IncomingMessage, ServerResponse, Server } from 'http';
+
+export type RequestHandler = (
+    req: IncomingMessage & {
+        params: {[key: string]: string}
+    },
+    res: ServerResponse
+) => any;
```

Comment:
I think you want Promise<void> here instead... I am looking at:  https://github.com/pedronauck/micro-router/blob/master/lib/index.js#L22 and https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/micro/index.d.ts#L16

Also it would be just better to use types from micro or extend those types than redefining them here.